remote.go

```diff
@@ -1124,6 +1134,35 @@ func (wd *remoteWD) Screenshot() ([]byte, error) {
 	return ioutil.ReadAll(decoder)
 }
 
+func (wd *remoteWD) WaitWithTimeoutAndInterval(condition Condition, timeout time.Duration, interval time.Duration) (error) {
+	startTime := time.Now()
+
+	for {
+		done, err := condition(wd)
+		if err != nil {
+			return err
+		}
+
+		if done {
+			return nil
+		}
+
+		elapsed := time.Since(startTime)
+		if elapsed > timeout {
+			return fmt.Errorf("Timeout after %v", elapsed)
```

Comment:
nit: errors should start with lower-case letters, when reasonable.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings